From 4de892c68058658e51c1b457038db46e5766587d Mon Sep 17 00:00:00 2001
From: fossdd <fossdd@pwned.life>
Date: Tue, 22 Jul 2025 15:18:27 +0200
Subject: [PATCH 11/11] sd-journal: remove last_stat

	In file included from ../src/libelogind/sd-journal/journal-internal.h:13,
	                 from ../src/libelogind/sd-journal/sd-journal.c:32:
	../src/libelogind/sd-journal/journal-file.h:80:21: error: field 'last_stat' has incomplete type
	   80 |         struct stat last_stat;
	      |                     ^~~~~~~~~

Signed-off-by: fossdd <fossdd@pwned.life>
---
 src/libelogind/sd-journal/journal-file.h | 1 -
 1 file changed, 1 deletion(-)

diff --git a/src/libelogind/sd-journal/journal-file.h b/src/libelogind/sd-journal/journal-file.h
index c3c5f5945..bb0030caf 100644
--- a/src/libelogind/sd-journal/journal-file.h
+++ b/src/libelogind/sd-journal/journal-file.h
@@ -77,7 +77,6 @@ typedef struct JournalFile {
         uint64_t last_n_entries;
 
         char *path;
-        struct stat last_stat;
         usec_t last_stat_usec;
 
         Header *header;
-- 
2.50.1

